### PR TITLE
Fix 'GetDescendants' method will case 'ArgumentException'

### PR DESCRIPTION
**Pull request by devkanro**
Fix 'GetDescendants' method get child count of 'start', it should be 'parent'.

### CODE REVIEW

**Comment by h82258652** [comment]
Good, I also create a demo to reproduce this bug.
https://github.com/h82258652/WinrtXamlToolkitVisualTreeHelperExtensionsBugDemo

File: WinRTXamlToolkit/Controls/Extensions/VisualTreeHelperExtensions.cs [hunk 1]
```diff
@@ -158,7 +158,7 @@ public static IEnumerable<DependencyObject> GetDescendants(this DependencyObject
                         }
```
Comment (xyzzer) [inline WinRTXamlToolkit/Controls/Extensions/VisualTreeHelperExtensions.cs:158]: One more "start" two lines up.

**Review by xyzzer — COMMENTED** [review]
Thanks! Wow, this bug lasted a while!

File: WinRTXamlToolkit/Controls/Extensions/VisualTreeHelperExtensions.cs (hunk 1 above)
Comment (h82258652) [reply WinRTXamlToolkit/Controls/Extensions/VisualTreeHelperExtensions.cs:158]: yes, ```start is DataTemplate``` should be ```parent is DateTemplate```.

**Comment by h82258652** [comment]
@xyzzer 
hope to see the fixed nuget package as soon as possible.

**Comment by xyzzer** [comment]
As soon as I have a moment

**Comment by xyzzer** [comment]
Done

**Comment by xyzzer** [comment]
It should be done now

**Comment by h82258652** [comment]
@xyzzer 
yes, it works for me now, thanks a million.

**Pull request merged**